### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?

Adding some CSS animations to the header and footer links/tabs/buttons.

Updated `keyline-grey` variable to work with the darker `cream`.

### CODE REVIEW

File: client/scss/components/_footer.scss [hunk 1]
```diff
@@ -112,6 +112,11 @@
   display: block;
   text-decoration: none;
   border-left: 1px solid color('charcoal');
+  transition: color 200ms ease;
```
Comment (jamesgorrie) [inline client/scss/components/_footer.scss]: Could we be locking our animations into vars too?

File: client/scss/components/_opening_hours.scss [hunk 2]
```diff
@@ -53,15 +59,19 @@
...
 
   .enhanced & {
-    display: none;
+    visibility: hidden;
+    height: 0;
+    overflow: hidden;
   }
 
   &.opening-hours__table--is-visible {
-    display: table;
```
Comment (gestchild) [inline client/scss/components/_opening_hours.scss]: I think removing this is stopping the table from being full width

**Comment by gestchild** [comment]
There seems to be some layout issues with the address now, looks like flexbox shenanigans

File: client/scss/components/_footer.scss (hunk 1 above)
Comment (davidpmccormick) [inline client/scss/components/_footer.scss]: I was wondering about this, too – quite possibly, yeah. Perhaps a PR to rationalise them throughout.

File: client/scss/components/_opening_hours.scss (hunk 2 above)
Comment (davidpmccormick) [inline client/scss/components/_opening_hours.scss]: Hmmmm… I added `width: 100%;` to account for this. Works on my machine(!)

**Comment by davidpmccormick** [comment]
Part of #193.

**Pull request merged**